Dockerfile

```diff
@@ -41,14 +41,28 @@ RUN apt-get update && \
   ffmpeg \
   libavdevice-dev && \
   rm -rf /var/lib/apt/lists/*
+# install required libs for building graphviz
+RUN apt-get update && \
```

Comment:
So much stuff for one library? What is graphviz used for? Plus, Why not just merge this above ?

Reply:
Plus don't install libpython or it's dev or any such thing. It interferes with the installed python already on the container